### PR TITLE
Set Up Redux ToolKit

### LINKED ISSUE #32: Ch:  Setup Project Redux
### Build a setup  Project  
 As developer i want to configure Project setup  for  E-commerce front End
### Task to be completed

- [ ] Setup Config.ts  to define  use  colors and fonts
- [ ] Configure Router Dom
- [ ] Configure Redux toolkit

### PR DESCRIPTION
**Pull request by 13XAVI**
### Overview
As a Frontend Developer, I want to set up a new frontend project with a well-structured foundation, so that it can be easily developed and maintained by the team.

### Acceptance Criteria
- [ ] Redux Toolkit is configured to manage the application's state efficiently.

### Completed Task Checklist

- [ ] Configure Redux Toolkit

By following these tasks and verifying the acceptance criteria, the frontend project will have a solid foundation, making it easier to develop and maintain.

### CODE REVIEW

**Review by wayneleon1 — APPROVED** [review]
Good job

**Review by jkarenzi — APPROVED** [review]

File: .eslintrc.cjs
Comment (dushimeemma) [inline .eslintrc.cjs:1]: Why have you changed this file?

File: src/Redux/EcommerceReducer.ts
Comment (dushimeemma) [inline src/Redux/EcommerceReducer.ts:1]: what is the use of this file in configuring redux?

File: src/index.css
Comment (dushimeemma) [inline src/index.css:1]: why have you changed this file while setting up redux?

File: src/main.tsx [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import { Provider } from 'react-redux';
+import store from './Redux/store';
 import App from '@/App.tsx';
 import '@/index.css';
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <Provider store={store}>
```
Comment (dushimeemma) [inline src/main.tsx]: Exchange lines 10 and 9.
ie: Line 10 should come first on line 9 and line 9 should be on line 10.
example:
`<React.StrictMode>
      <Provider store={store}>
            <App />
      </Provider>
</React.StrictMode>`

**Review by dushimeemma — CHANGES_REQUESTED** [review]

**Pull request closed without merging**